### PR TITLE
Issue 772 parameters

### LINKED ISSUE #772: Implement different electrolyte functions
From Valoen et al ; http://jes.ecsdl.org/content/152/5/A882

### PR DESCRIPTION
**Pull request by tinosulzer**
# Description

Adds parameter set Xu2019.
Some small reformatting of the half-cell model
Fixes #772 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Comment by tinosulzer** [comment]
Requires #1514

File: pybamm/input/parameters/lithium_ion/separators/separator_Xu2019/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Separator parameters
+
+Parameters for a ??? separator, from the paper
```
Comment (brosaplanella) [inline pybamm/input/parameters/lithium_ion/separators/separator_Xu2019/README.md:3]: Do you want to leave the ??? or was that a placeholder?

**Review by brosaplanella — APPROVED** [review]
Looks good, thanks Tino!

File: pybamm/input/parameters/lithium_ion/separators/separator_Xu2019/README.md (hunk 1 above)
Comment (tinosulzer) [reply pybamm/input/parameters/lithium_ion/separators/separator_Xu2019/README.md]: placeholder, good spot

**Pull request merged**